src/frontend/App.tsx

```diff
@@ -40,6 +40,7 @@ import { UpdateModalController } from './components/UI/UpdateModalController'
 import { QuestsPage } from './screens/Quests'
 import { NavigateListener } from './NavigateListener'
 import G7Webview from './screens/G7Webview'
+import CardPrivacyPolicy from './screens/Onboarding/analytics/CardPrivacyPolicy'
```

Comment:
I believe this is because the folder is named `cardPrivacyPolicy`. Can we uppercase it since it is a react component to be `CardPrivacyPolicy`?